packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php

```diff
@@ -2,1100 +2,1100 @@
     <!-- Title of the page -->
     <x-slot:title>
         @lang('admin::app.marketing.promotions.cart-rules.create.title')
-    </x-slot>
+        </x-slot>
 
-    {!! view_render_event('bagisto.admin.marketing.promotions.cart_rules.create.before') !!}
+        {!! view_render_event('bagisto.admin.marketing.promotions.cart_rules.create.before') !!}
 
-    <v-cart-rule-create-form />
+        <v-cart-rule-create-form />
 
-    {!! view_render_event('bagisto.admin.marketing.promotions.cart_rules.create.after') !!}
+        {!! view_render_event('bagisto.admin.marketing.promotions.cart_rules.create.after') !!}
 
 
-    @pushOnce('scripts')
-        <!-- v cart rules create form template -->
-        <script
-            type="text/x-template"
-            id="v-cart-rule-create-form-template"
-        >
-            <div>
-                <x-admin::form
-                    :action="route('admin.marketing.promotions.cart_rules.store')"
-                    enctype="multipart/form-data"
-                >
+        @pushOnce('scripts')
+            <!-- v cart rules create form template -->
+            <script
+                type="text/x-template"
+                id="v-cart-rule-create-form-template"
```

Comment:
@claudioghirellifh 
Here, some extra tab spaces are coming. Please remove the extra spaces.

Reply:
@amit-webkul 
Using PhpStorm, it did not flag these extra spaces for me. Now, I performed the formatting using VSCode for .blade files.
![2024-01-31_09-40](https://github.com/bagisto/bagisto/assets/42969756/f3d6b374-9c6b-443a-8b74-1ff1e2833575)